company_website/templates/main_page_partials/testimonials.haml

```diff
@@ -1,2 +1,49 @@
-%br
-  testimonials
\ No newline at end of file
+{% load static %}
+
+%link{:rel => "stylesheet",
+    :type => "text/css",
+    :href => "https://cdnjs.cloudflare.com/ajax/libs/OwlCarousel2/2.3.4/assets/owl.carousel.min.css",
+    :integrity => "sha384-kcNNzf7wI8//ZkNUaDd5JwxLoFaBgkj9Z4O4NwtuX9Lkmsz0HoITOxJsGkYxDuyG",
+    :crossorigin => "anonymous"}
+
+%link{:rel => "stylesheet",
+    :type => "text/css",
+    :href => "https://cdnjs.cloudflare.com/ajax/libs/OwlCarousel2/2.3.4/assets/owl.theme.default.min.css"
+    :integrity => "sha384-8/AzSKHReNfZT4HGFTyRyJ1jXhhx/ZMnmX08Cy6TeaKWj0Vzho0AabG06C8mh02i",
+    :crossorigin => "anonymous"}
```

Comment:
I think that we should keep those links in head

Reply:
That's correct, but at the time it wasn't known where exactly to store them, due the way main template was redesigned. Thus, they were stored here until sensible solution would be found.

Reply:
Done.